### ISSUE TITLE
Pointer jumps if you are holding any key on the tablet, and bugs with Stylus (H640P)

### ISSUE THREAD

**Issue by JumperLuko** [DIGImend/digimend-kernel-drivers#530]
I'm using Linux Mint 20, tested on Gnome (Xorg and Wayland) and Cinnamon.
While I'm holding a key on the tablet and letting go of the pen (still pressing the key), the mouse pointer moves up to the left.

I clicked on reinstalling the v10 driver (.deb version) without changes.
And as you can see in the video, testing the pressure on the Gnome panel has stopped working.
And in the menu with the pen, pressing and releasing another option does not work (in the video where I change the table from absolute to relative), with a mouse it works

In short, something about the identification of the pen is not quite right, possibly a configuration.

For me who uses the keys as a control and shift in Blender, it is very frustrating.

https://user-images.githubusercontent.com/22037309/115638505-eb604780-a2e8-11eb-80eb-b3721f1a98e7.mp4

**Comment by JumperLuko** [comment]
I tested the table on the Linux Mint USB Live and the bug of jumping the mouse pointer is the same, even without this driver installed.
Is it a conflict of input that cannot be solved? I imagine that the table is not made to use the stylus while pressing the button, designed to work just to click.

**Comment by JumperLuko** [comment]
## Gnome little fixes
On Wayland: 
- testing the pressure on the Gnome panel is working 
-  select an option on menus works: click and click on option.
